config/params/basis_points.go

```diff
@@ -0,0 +1,10 @@
+package params
+
+import "github.com/OffchainLabs/prysm/v6/consensus-types/primitives"
+
+const BASIS_POINTS = primitives.BP(10000)
```

Comment:
I don't think we use `ALL_CAPS` for any constant in the codebase, it's usually `CamelCase`